external-crates/move/crates/move-compiler/tests/move_check/to_bytecode/clever_assertions.move

```diff
@@ -0,0 +1,25 @@
+module 0x42::m {
```

Comment:
Do we really need these in move-check too?

Reply:
I think so? Since I want to make sure we get the correct/expected error behavior in these cases. Although, this is probably better placed under `typing/...`

Reply:
I don't see why we need to duplicate it though? Like we can just test that it is correctly feature gated and then bail